File: RP4/d3.java
package RP4;

import java.util.Scanner;

public class d3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] A = new int[15];
        int[] B = new int[15];

        for (int i = 0; i < 15; i++) {
            System.out.print("Digite o número " + (i + 1) + ": ");
            A[i] = scanner.nextInt();
        }

        for (int i = 0; i < 15; i++) {
            B[i] = A[i] * A[i];
        }

        System.out.println("\nMatriz A:");
        for (int i = 0; i < 15; i++) {
            System.out.print(A[i] + " ");
        }

        System.out.println("\n\nMatriz B (quadrados):");
        for (int i = 0; i < 15; i++) {
            System.out.print(B[i] + " ");
        }

        scanner.close();
    }
}

File: RP4/d3.py
A = []
B = []

for i in range(15):
    num = int(input(f"Digite o número {i+1}: "))
    A.append(num)

for i in range(15):
    B.append(A[i] ** 2)

print("\nMatriz A:")
print(A)

print("\nMatriz B (quadrados):")
print(B)